php/util/class-fieldmanager-util-validation.php

```diff
@@ -5,6 +5,9 @@
  * @package Fieldmanager_Util
  */
 
+/**
+ * Class for Fieldmanager_Util_Validation.
+ */
```

Comment:
This sorta seems like it should be the file header, and the file header the class description.